int32 flag
float64 x
float64 y
float64 z
float64 q_w
float64 q_x
float64 q_y
float64 q_z
---
float64 angle0
float64 angle1
float64 angle2
float64 angle3
float64 angle4
float64 angle5
float64 angle6
